backend/data/scrapers/payload.py

```diff
@@ -26,70 +86,26 @@ def create_payload(size, content_type, year = LIVE_YEAR):
         year = LIVE_YEAR
 
     return {
-        "query": {
-            "bool": {
-                "must": [
-                    {"term": {"live": True}},
-                    [
-                        {
-                            "bool": {
-                                "minimum_should_match": "100%",
-                                "should": [
-                                    {
-                                        "query_string": {
-                                            "fields": [f"{content_type}.implementationYear"],
-                                            "query": f"*{year}*",
-                                        }
-                                    }
-                                ],
-                            }
-                        },
-                        {
-                            "bool": {
-                                "minimum_should_match": "100%",
-                                "should": [
-                                    {
-                                        "query_string": {
-                                            "fields": [f"{content_type}.studyLevelValue"],
-                                            "query": "*ugrd*",
-                                        }
-                                    }
-                                ],
-                            }
-                        },
-                        {
-                            "bool": {
-                                "minimum_should_match": "100%",
-                                "should": [
-                                    {
-                                        "query_string": {
-                                            "fields": [f"{content_type}.active"],
```

Comment:
do we want to add back the active query field?

Reply:
i will investigate what adding it back will do :monocle_face:

Reply:
I put it in in the right format, and it gave 0 results, so I'm just gonna leave it out and assume it doesn't work anymore